compliance/collection/nodescanv2/nodescan.go

```diff
@@ -0,0 +1,25 @@
+package nodescanv2
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
```

Comment:
Cosmetics and subjective, bug I think, `log` should go to `_impl.go` file if that exists. In this case this is `fake_nodescan.go`.
Log is an implementation detail that doesn't have to be exposed in the file where you define interfaces.

Reply:
Well spotted, that was a leftover from refactoring. Moved!